docs/snippets/react/table-story-fully-customize-controls.js.mdx

```diff
@@ -1,18 +1,36 @@
 ```js
 // Table.stories.js | Table.stories.jsx
 
-const TableStory = ({ data, ...args }) => (
-  <Table {...args} >
-    {data.map(row => (<TR>{row.map(item => <TD>{item}</TD>}</TR>))}
-  </Table>
-)
+import React from 'react';
 
-export const Numeric = TableStory.bind({});
+import Table from './Table';
+import TD from './TableDataCell';
+import TR from './TableRow';
```

Comment:
Do we want to do default imports here? Or switch them to named? (same applies for all .js, .mdx. ts versions for this file)

Reply:
I'll leave it as it is if you don't mind, as per the comment above. MDX/TS snippets will come at a later time as there's still some work to be done around it.